languages/python/exercises/concept/string-methods/.docs/hints.md

```diff
@@ -0,0 +1,21 @@
+## General
+
+[Introduction to string methods in Python][string-method-docs]
+
+## 1. Capitalize the title of the paper
+
+- You can use [string methods][string-method-docs] to capitalize the title properly.
```

Comment:
The way you've described the hint is perfect (not giving away the solution), but is it perhaps possible to directly link to the correct method? 

This comment also applies to the other hints.